packages/expo-modules-core/src/registerWebModule.ts

```diff
@@ -4,18 +4,21 @@ import type { NativeModule } from './ts-declarations/NativeModule';
 /**
  * Registers a web module.
  * @param moduleImplementation A class that extends `NativeModule`. The class is registered under `globalThis.expo.modules[className]`.
+ * @param moduleName – a name to register the module under.
```

Comment:
```suggestion
 * @param moduleName – a name to register the module under `globalThis.expo.modules[className]`.
```

nit: unfinished sentence?